docs/supported_apis/series_supported.rst

```diff
@@ -118,25 +125,27 @@ the related section on :doc:`Defaulting to pandas </supported_apis/index>`.
+| ``drop``                    | Y                               | **Omnisci**: ``D``                                 |
 +-----------------------------+---------------------------------+----------------------------------------------------+
 | ``drop_duplicates``         | Y                               |                                                    |
 +-----------------------------+---------------------------------+----------------------------------------------------+
 | ``droplevel``               | Y                               |                                                    |
 +-----------------------------+---------------------------------+----------------------------------------------------+
-| ``dropna``                  | Y                               |                                                    |
+| ``dropna``                  | Y                               | **Omnisci**: ``P`` since ``thresh`` and ``axis``   |
+|                             |                                 | params unsupported                                 |
 +-----------------------------+---------------------------------+----------------------------------------------------+
-| ``dt``                      | Y                               |                                                    |
+| ``dt``                      | Y                               | **Omnisci**: ``P``, only ``year``, ``month``,      |
+|                             |                                 | ``day`` and ``hour`` supported, otherwise ``D``    |
 +-----------------------------+---------------------------------+----------------------------------------------------+
 | ``dtype``                   | Y                               |                                                    |
 +-----------------------------+---------------------------------+----------------------------------------------------+
-| ``dtypes``                  | Y                               |                                                    |
+| ``dtypes``                  | Y                               | **Omnisci**: ``Y``                                 |
 +-----------------------------+---------------------------------+----------------------------------------------------+
 | ``duplicated``              | Y                               |                                                    |
 +-----------------------------+---------------------------------+----------------------------------------------------+
 | ``empty``                   | Y                               |                                                    |
 +-----------------------------+---------------------------------+----------------------------------------------------+
-| ``eq``                      | Y                               |                                                    |
+| ``eq``                      | Y                               | **Omnisci**: ``Y``                                 |

```

Comment:
don't forget to add a "see `add` for all engines" here when you'll add [this comment](https://github.com/modin-project/modin/pull/4225#discussion_r837233771) for Series' `add`

Reply:
I think we don't need `for all engines` comment. It is redundant. All info is in the description for `add` method. I created a PR addressing my comments - https://github.com/intel-ai/modin/pull/120.

Reply:
fillna restrictions are for Series as well?

Reply:
yep